packages/next/client/image.tsx

```diff
@@ -291,6 +320,17 @@ export default function Image({
   const heightInt = getInt(height)
   const qualityInt = getInt(quality)
 
+  const MIN_IMG_SIZE_FOR_PLACEHOLDER = 100
+  const tooSmallForBlurryPlaceholder =
+    widthInt && heightInt
+      ? widthInt * heightInt < MIN_IMG_SIZE_FOR_PLACEHOLDER
+      : false
+  const shouldShowBlurryPlaceholder =
+    configPlaceholder === PlaceholderValue.BLURRY &&
+    placeholder === PlaceholderValue.BLURRY &&
+    priority &&
```

Comment:
We only show blurry placeholders for priority images in order to avoid increasing initial load sizes for image-heavy pages. We should revisit this later after we've gotten some feedback from users.

Reply:
@styfle I remember you commented on this topic in the original RFC doc as well - I replied but I don't think you had a chance to follow up. Happy to discuss further concerns here if you have any.

Reply:
As per discussion in slack, I've updated the feature to be enabled for all images instead of just priority images (if the experimental feature is turned on).